.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install: 
+  pip install -r pip-requirements.txt
+
+# command to run tests
+script: 
+  nosetests -v --with-coverage --cover-package=presenters,plotting.figuremanager
```

Comment:
I strongly suggest fixing the mimimum required coverage to be at your current coverage level (at least) so that if there's a drop in coverage the CI will break on future PR requests. 
nose tests has a `--cover-min-percentage` option that you can specify. 

This will be crude mechanism of ensuring that future developers and maintainers are forced to keep the test coverage at a mimimum standard.

I also think that the `--cover-branches` option should be added, but that's probably best left for another PR.